### ISSUE TITLE
Differences in reconstructed images

### ISSUE THREAD

**Issue by rasad-hub** [CERN/TIGRE#601]
Hello. I tried to reconstruct the sinogram matrix I got from resolusion-phantom through tigre algorithms and I am sure that I enter the settings of the parameters correctly, but the reconstructed image is different from the image I expect. The photo on the left is related to the same sinogram.  And it is reconstructed by a simple command (iradon) and the image on the right is reconstructed by tigre algorithms. Can you guide me, where is the problem?

  - MATLAB version:2023b
  - OS:windows10
  - CUDA version:11.8

**Comment by AnderBiguri** [comment]
The only guess I can give you is that you have set the geometry wrong somehow. It also seems that you have not log-transformed your projections either (on both cases).

**Comment by rasad-hub** [comment]
> The only guess I can give you is that you have set the geometry wrong somehow. It also seems that you have not log-transformed your projections either (on both cases).

the code i used:
--------------------------------------------------------------------
geo.nDetector=[300;  1];					% number of pixels              (px)


angles=linspace(0,2*pi,180);
a = single(load my 2D 180*300 sinogram);
projections = permute(a,[3,2,1]);
img = FBP(projections,geo,angles);
------------------------------------------------------------------------------------------
i have written the parts that i have changed, it is possible that i have made a mistake somewhere?

**Comment by AnderBiguri** [comment]
yes, as I don't know if the angles match the measured data, or the geometry is right, it should match what the real thing is, which I don't know. Maybe try negating the angles, or other things.

**Comment by AnderBiguri** [comment]
@rasad-hub no, not at all. Both of your recons look wrong anyway, probably the log-transform, as I said. But TIGRE should work better than matlabs iradon.
